pylib/cqlshlib/test/test_cqlsh_completion.py

```diff
@@ -473,7 +473,7 @@ def test_complete_in_delete(self):
         self.trycompletions('DELETE FROM twenty_rows_composite_table USING TIMESTAMP 0 WHERE TOKEN(a ',
                             choices=[')', ','])
         self.trycompletions('DELETE FROM twenty_rows_composite_table USING TIMESTAMP 0 WHERE TOKEN(a) ',
-                            choices=['>=', '<=', '=', '<', '>'])
+                            choices=['>=', '<=', '=', '<', '>', '!='])
```

Comment:
We do not support `!=` on token restrictions. NEQ stands with other eq operators in Parser.g, allowed on token and because we allow all binary operators with token functions, removing below from `Relation#toRestriction(TableMetadata table, VariableSpecifications boundNames)`:
```
if (operator == Operator.NEQ)
            throw invalidRequest("Unsupported '!=' relation: %s", this);
```
led to allowing it for token functions. 
I will special case the `!=` for tokens:
```
if (operator == Operator.NEQ && columnsExpression.kind() == ColumnsExpression.Kind.TOKEN)
            throw invalidRequest("Unsupported '!=' relation: %s", this);
```
Thank you for catching it!!